s4e-web/cypress/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "strict": true,
+    "strictNullChecks": false,
```

Comment:
If you're using some walk around's write why you're using it. The shorter and simpler description, the better.

Reply:
finally I use a different method so i deleted it